app/src/main/java/tech/relaycorp/gateway/domain/LocalConfig.kt

```diff
@@ -74,11 +93,10 @@ class LocalConfig
         )
     }
 
-    private suspend fun generateCargoDeliveryAuth(idKeyPair: IdentityKeyPair) {
-        val cda = selfIssueCargoDeliveryAuth(
-            idKeyPair.privateKey,
-            idKeyPair.certificate.subjectPublicKey
-        )
+    private suspend fun generateCargoDeliveryAuth() {
+        val key = getIdentityKey()
+        val certificate = getIdentityCertificate() ?: return
```

Comment:
Is that `return` ever executed? I think `getIdentityCertificate()` always returns a value.